### DISCUSSION TITLE
Show all callers of a method, across repos

### DISCUSSION THREAD

**Discussion by melissachang** [community/community#9683] [Other Feature Feedback, Questions, & Ideas] [answered]
I want to know who calls `datasets().create():`. I'd like to be able to click on it and show callers, from all all repos.

![image](https://user-images.githubusercontent.com/10929390/148308628-6cbcf057-8f44-4376-b7f8-58b09bc9464f.png)

**Comment by jglick** [comment, marked as answer]
Duplicate of #8593?
